homeassistant/components/mqtt/discovery.py

```diff
@@ -41,6 +41,86 @@
 ALREADY_DISCOVERED = 'mqtt_discovered_components'
 MQTT_DISCOVERY_UPDATED = 'mqtt_discovery_updated_{}'
 
+TOPIC_PREFIX = '~'
+
+ABBREVIATIONS = {
+    'aft': 'after',
+    'arm': 'arm',
+    'attr': 'attributes',
+    'aux': 'aux',
+    'avty': 'availability',
+    'avail': 'available',
+    'away': 'away',
+    'bri': 'brightness',
+    'cla': 'class',
+    'cls': 'close',
+    'clsd': 'closed',
+    'code': 'code',
+    'clr': 'color',
+    'cmd': 'command',
+    'curr': 'current',
+    'dev': 'device',
+    'disarm': 'disarm',
+    'fx': 'effect',
+    'exp': 'expire',
+    'fan': 'fan',
```

Comment:
We should be able to make the code in such a way that if the key is not found, it doesn't translate it and so stays the same. Allows us to remove all entries for which there is no translation.